documentation/modules/tracing/proc-enabling-tracing-type.adoc

```diff
@@ -0,0 +1,53 @@
+// Module included in the following assemblies:
+//
+// assembly-distributed tracing.adoc
+
+[id='proc-enabling-tracing-type-{context}']
+= Introducing a different OpenTelemetry tracing system
+
+[role="_abstract"]
+Instead of the default Jaeger system, you can specify other tracing systems that are supported by OpenTelemetry.
+You do this by adding the required artifacts to the library of the Kafka image provided with Strimzi.
```

Comment:
"to the libs folder"? or just "adding the requered artifacts to the Kafka image ..."